src/extras/exporters/gltf-exporter.js

```diff
@@ -472,6 +478,12 @@ class GltfExporter extends CoreExporter {
                 const entityMeshInstance = resources.entityMeshInstances.find(e => e.node === entity);
                 if (entityMeshInstance) {
                     node.mesh = resources.entityMeshInstances.indexOf(entityMeshInstance);
+
+                    // Add skin reference if this node has a skinned mesh
+                    const meshInstance = entityMeshInstance.meshInstances[0];
```

Comment:
Slight problem here is that the engine supports different skin for each mesh instance in the array here, but glTF does not.
We should check all meshInstances in this array, and they must point to the same skin. If they point to different skins, or some do not link to any skin, we should probably log a warning and skip skinning.

Reply:
Hmm. Any objection to opening a new issue on this and fixing it in a follow up PR? I want to keep this PR reasonably simple and there might be some nuance to how we want to deal with the case you mention.